cypress/integration/user_can_sign_up_spec.js

```diff
@@ -2,7 +2,7 @@ const signUp = require('../support/signup_helper')
 
 describe("Registration", () => {
   it("A user signs up and is redirected to sign in", () => {
-    signUp("username", "password", "email@email.com");
+    signUp("email@email.com", "password", "username");
     cy.url().should("include", "/sessions/new");
```

Comment:
Should this be include, or should it be eq?

Reply:
Changed to eq